dbms/src/Common/UInt128.h

```diff
@@ -25,19 +24,43 @@ struct UInt128
     UInt64 first;
```

Comment:
Renaming this fields to `high` and `low` (or, better, `low` and `high`?) will be more convenient.